pkg/build/make.go

```diff
@@ -83,12 +83,6 @@ func (m *Make) MakeCross(version string) error {
 		return fmt.Errorf("checking out version %s: %w", version, err)
 	}
 
-	// Unset the build memory requirement for parallel builds
-	// TODO: Remove this once the 1.20 release reaches EOL.
-	const buildMemoryKey = "KUBE_PARALLEL_BUILD_MEMORY"
```

Comment:
Hm, this is the memory required for building in parallel according to:

https://github.com/kubernetes/kubernetes/blob/4eb6b3907a68514e1b2679b31d95d61f4559c181/hack/lib/golang.sh#L917-L929

So I assume that the change will have no effect on how we build k8s (in parallel). We would have to set it to a higher number to enforce serial build I guess.